documentation/modules/proc-deploy-service-monitors.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// assembly-monitoring-kube.adoc
+// assembly-monitoring-oc.adoc
+
+[id='deploy-service-monitors-{context}']
+= Deploying Service Monitors
+
+Follows these steps to enable Prometheus to monitor {ProductName}.
+
+.Prerequisites
+
+* Monitoring infrastructure must be present, either pre-existing or installed using the above steps. 
```

Comment:
Change to: Monitoring infrastructure must be present, whether pre-existing or link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#deploy-monitoring-infrastructure-messaging[installed following this procedure].